kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -296,7 +296,10 @@ with DAG(
     {% endif %}
 
     {% if not config.run_config.volume.disabled %}
-    kedro_group >> delete_pipeline_storage
```

Comment:
Any reason to remove this dependency? Pipeline storage shouldn't be deleted before `kedro_group` ends

Reply:
Ouch, good spot, fixing.